arches/app/media/js/views/components/cards/photo-gallery-card.js

```diff
@@ -48,13 +49,19 @@ define([
 
         this.fileListNodeId = getfileListNode();
 
+        this.cleanUrl = function(url) {
+            const httpRegex = /^https?:\/\//;
+            return !url || httpRegex.test(url) || url.startsWith(arches.urls.url_subpath) ? url :
+                (arches.urls.url_subpath + url).replace('//', '/');
+        };
+
         this.getUrl = function(tile){
             var url = '';
             var name = '';
             var val = ko.unwrap(tile.data[this.fileListNodeId]);
             if (val && val.length == 1) {
                 {
-                    url = ko.unwrap(val[0].url) || ko.unwrap(val[0].content);
+                    url = self.cleanUrl(ko.unwrap(val[0].url)) || ko.unwrap(val[0].content);
```

Comment:
I guess it makes sense not to bother cleaning the content, if it's expected to be a blob.